templates/bot-msgext/js/default/dialogs/rootDialog.js

```diff
@@ -67,7 +67,7 @@ class RootDialog extends ComponentDialog {
                 return await innerDc.cancelAllDialogs();
             }
             default: {
-                if (
+                if (text &&
```

Comment:
Can we put text not null check out of this block? right after trim()?

Reply:
OK